src/cc65/symtab.c

```diff
@@ -728,10 +743,17 @@ SymEntry* AddLabelSym (const char* Name, unsigned Flags)
                 /* We're processing a label, let's update all gotos encountered
                 ** so far
                 */
+                SymEntry *E;
                 g_userodata();
                 g_defdatalabel (DOR->LateSP_Label);
                 g_defdata (CF_CONST | CF_INT, StackPtr - DOR->StackPtr, 0);
 
+                /* Optimizer will need the information about the value of SP adjustment
+                ** later, so let's preserve it. */
```

Comment:
The predominant style wants to have the comment closing of multi-line comments on its own line.